Task2/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите радиус окружности");
+            int a = Convert.ToInt32(Console.ReadLine());
```

Comment:
Неверный выбор типа переменной. Почему радиус должен быть только целым числом?
Кроме того, постарайтесь избегать использования методов класса `Convert` без особой необходимости. Пользуйтесь методами имеющихся типов.
```suggestion
            var a =double.Parse(Console.ReadLine());
```